doc_src/interactive.rst

```diff
@@ -134,7 +134,7 @@ Variable                                                   Meaning
 ``fish_color_search_match``                                history search matches and selected pager items (background only)
 ==========================================                 =====================================================================
 
-If a variable isn't set, fish usually tries ``$fish_color_normal``, except for:
+If a variable isn't set or is empty, fish usually tries ``$fish_color_normal``, except for:
```

Comment:
sounds like a good idea (assuming that not too many themes rely on the empty=normal behavior)

I'm not sure how it enables themes to override variables but I didn't think very hard..

Reply:
What I mean by that is that this allows making a theme with only global variables, and it will override any potentially set universal variables. 

So you can start by setting all variables to empty and then set the ones you want, you don't have to *erase* variables (which for uvars would be permanent).

>assuming that not too many themes rely on the empty=normal behavior

Of *our* themes only the None theme uses any empty values, and that sets the fallback colors to normal anyway.

I don't imagine this is used much elsewhere because I don't think there is much demand for the "normal" color. Most themes, in my experience, look like [dracula.fish](https://github.com/dracula/fish/blob/master/conf.d/dracula.fish) - a fixed palette of RGB colors.

It's also very simple to fix: If you notice your theme colors being not-normal where you expected normal, simply add a "normal" where you had an empty value. This is even backwards-compatible, *and* it is more like what you would get with `set_color`.

`set_color $fish_color_option $fish_color_param` won't be normal only because $fish_color_option isn't set.

Reply:
sounds good, thanks. The new behavior is more intuitive